lib/spark/dsl/entity.ex

```diff
@@ -52,6 +52,28 @@ defmodule Spark.Dsl.Entity do
     docs: ""
   ]
 
+  alias Spark.Dsl.Entity
+
+  @type t :: %Entity{
+          name: atom,
+          target: module,
+          transform: mfa,
+          recursive_as: atom,
+          examples: [String.t()],
+          entities: [t],
+          deprecations: [{atom, String.t()}],
+          describe: String.t(),
+          snippet: String.t(),
+          args: [atom],
+          links: [{atom, [String.t()]}],
+          hide: boolean,
+          modules: [String.t()],
+          no_depend_modules: [atom],
+          schema: NimbleOptions.schema(),
```

Comment:
These should probably use `Spark.OptionsHelpers.schema()`, right?

Reply:
good catch!